### ISSUE TITLE
Question about Jacobian determinant in GNF loss function

### ISSUE THREAD

**Issue by pigu163** [johncava/GNF-pytorch#4]
Hello,

I've been studying your Graph Normalizing Flow implementation and noticed that while the code calculates the log determinant of the Jacobian (`log_det_xz` and `log_det_zx`), these terms don't appear to be included in the final loss function:

```python
# Current loss only includes link prediction
pos_loss = -torch.log(y).mean() 
neg_loss = -torch.log(1 - y + 1e-15).mean()
loss = neg_loss + pos_loss
```

In normalizing flows, shouldn't the loss also include the Jacobian term and the base distribution's negative log-likelihood?

I was wondering if this was intentional or if I'm missing something in my understanding.
